security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,52 @@ These release notes track the development of the Compliance Operator in the {pro
+
+* A `PriorityClass` can now be set through `ScanSetting`. This will ensure the Compliance Operator is prioritized and minimizes the chance that the cluster will fall out of compliance. For more information, see xref:../../security/compliance_operator/compliance-operator-advanced.adoc#compliance-priorityclass_compliance-advanced[Setting `PriorityClass` for `ScanSetting` scans].
+
+[id="compliance-operator-0-1-55-bug-fixes"]
+=== Bug fixes
+
+* Previously, the Compliance Operator hard-coded notifications to the default namespace. If the Operator were installed in a non-default namespace, the notifications would not work as expected. Now, Notifications work in non-default namespaces. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060726[*BZ#2060726*])
+
+* Previously, the Compliance Operator was unable to evaluate default configurations used by kubelets, resulting in inaccurate results and false positives. This new feature evaluates the kubelet configuration and now reports accurately. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2075041[*BZ#2075041*])
+
+* Previously, the Compliance Operator had a set rule for speed at which evaluations were recorded, causing the `ocp4-kubelet-configure-event-creation` rule to fail even after remediations were applied. Now, the check and remediation uses the same variable to evaluate and remediate the system. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2082416[*BZ#2082416*])
+
+* The `ocp4-configure-network-policies` rule requires manual intervention to perform effectively. New descriptive instructions and rule updates increase applicability of the `ocp4-configure-network-policies` rule for clusters using Calico CNIs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2091794[*BZ#2091794*])
+
+* Previously, the Compliance Operator would not clean up pods used to scan infrastructure when using the `debug=true` option in the scan settings. This caused pods to be left on the cluster even after deleting the Scan Setting Binding. Now, pods are always deleted when a Scan Setting Binding is deleted.(link:https://bugzilla.redhat.com/show_bug.cgi?id=2092913[*BZ#2092913*])
+
+* Previously, the Compliance Operator used an older version of `operator-sdk` that caused alerts about deprecated functionality. Now, an updated version of `operator-sdk` is included and there are no more alerts for deprecated functionality. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2098581[*BZ#2098581*])
+
+* Previously, the Compliance Operator made assumptions about machine configuration names, and as a result was unable to determine the relationship between machine configurations and kubelet configurations. Now, the Compliance Operator has improved handling of the machine configurations and is able to determine if a kubelet configuration is a subset of a machine configuration. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2102511[*BZ#2102511*])
```

Comment:
This one does a good job describing the technical details, but I wonder if we should include what made this a bug. As a reader, I'm not sure how this affects my usage of the Compliance Operator (e.g., I should have been more clear in my doc text!)

Perhaps something like:

```
Previously, the Compliance Operator would fail to apply remediations if it couldn't
determine the relationship between kubelet and machine configurations....
```